docs/components/arm/fake.md

```diff
@@ -82,17 +83,19 @@ Refer to the following JSON examples for differences in configuration between th
     "type": "arm",
     "model": "fake",
     "attributes": {
-        "model-path": "/Users/<YOUR-USERNAME>/downloads/universalrobots/ur5e.json"
+        "model-path": "</Users/<YOUR-USERNAME>/downloads/universalrobots/ur5e.json>"
     }
 }
 ```
 
 {{% /tab %}}
 {{< /tabs >}}
 
-Once you have successfully configured your `fake` arm, you can navigate to the **control** tab of [the Viam app](https://app.viam.com).
-A drop-down menu should appear with the name of your arm that allows you to toggle the fake arm's joint positions and Cartesian end positions to dynamically test motion planning:
+Once you have successfully configured your `fake` arm, navigate to the **control** tab of [the Viam app](https://app.viam.com).
+A drop-down menu appears with the `name` of your arm.
 
-<img src="../img/fake-arm-ui-remote-control.png" alt="Motion planning remote-control of a fake ur5e arm in the Viam app control tab." style="max-width:800px" />
+Open this interface to adjust the `fake` arm's joint positions and Cartesian end positions:
+
+![Motion planning remote-control of a fake ur5e arm in the Viam app control tab](../img/fake-arm-ui-remote-control.png)
 
 Note that this visual example sets the `fake` arm to act as a  `ur5e` arm.
```

Comment:
I think delete all of this? We don't do this for other arm models, or for other fake models, and people can refer to the mock robot tutorial for this info. Additionally, this doesn't appear unless the robot is live, so if this must stay in I think add a note to that affect.

Reply:
Do you think most people will know to go to or have read the mock robot tutorial? We could try linking between, but I'm not sure if I want to remove documentation of this function from here

Reply:
I'm just not sure why this would be included on this specific model page when it isn't in any other model pages. The control tab works for all models, and we document the control tab in various fleet management docs as well as in tutorials, so I wouldn't think this needs to be here. Is there a reason that this particular page is special?

Reply:
@JessamyT Good point, I hadn't realized that the same degree of mock control was available in the control tab for other fakes. I removed here-- I think this is more motivation to document the control tab capabilities better and maybe return to documenting this function in all `fake` models (or at least pointing to a place where we do) in the near future? 
bc I agree there's no need to document here and not elsewhere, but the control tab is a cool feature for so many components that we have yet to talk about that much on component pages!